docs/terminal/adelaide.md

```diff
@@ -232,7 +232,9 @@ When the aircraft is ready for takeoff, **EDN ADC** will give a "Next" call, whe
     <span class="hotline">**EDN ADC** -> **AD TCU**</span>: "A050, BLKT11"
 
 #### Arrivals
-AD TCU Must coordinate the sequence information to **EDN ADC** prior to transfer of jurisdiction.
+AD TCU, Should assign the appropriate *STAR* (**AD**,**ATNAR**,,**BEVSO**,**KAKLU**,**KLAVA**)
+
+AD TCU Must coordinate the sequence information to **EDN ADC** prior to transfer of jurisdiction. 
 
 !!! phraseology
     <span class="coldline">**AD TCU** -> **EDN ADC**</span>: "New sequence of 2. MRNR21, via AD, Number 1. MRNR22, via AD, Number 2"  
```

Comment:
ED only owns up to A020, while approaches commence at A032/A033. AD coordinates the requested approach to ED, who clears the aircraft through AD TCU, who then relays it to the pilot.

eg:

`AD TCU: 10 to the NW that is MRNR11, requesting ILSY apch RWY 18`
`ED ADC: cleared ILS apch rwy18 mrnr11`
`AD TCU: cleared ILS apch rwy18 mrnr11`

`AD TCU -> MRNR11: mrnr11 cleared ILS apch rwy18 contact ed tower 118.3`

Reply:
This sounds similar to YSBK where ADC provides clearance to TCU to provide approach clearance. I think we simplified it into heads up coord to make it a bit more relevant to VATSIM